packages/stable/src/exports.gen.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Cognite AS
+export {} from './types.gen';
```

Comment:
Ah. It's because code has only been generated for only one service. This gets populated as soon as we have codegen for another service, so I'm not sure it's worth the extra effort of making generated code nice when only one service has it enabled.